### PR TITLE
In trades replace BID / ASK with BUY / SELL.

### PR DESCRIPTION
**Pull request by ooinaru**
Primarily semantics changes, but I feel like trades side should either be BUY or SELL instead of BID / ASK, which is used when describing the order book. 
Also, side on Binance can be had from side if buyer is maker.

### CODE REVIEW

File: cryptofeed/binance/binance.py [hunk 1]
```diff
@@ -60,7 +60,7 @@ def __reset(self):
         await self.callbacks[TRADES](feed=self.id,
                                      order_id=msg['t'],
                                      pair=pair_exchange_to_std(msg['s']),
-                                     side=UND,
```
Comment (bmoscon) [inline cryptofeed/binance/binance.py]: if you are no longer using UND you should remove it from the imports

**Comment by bmoscon** [comment]
also please update the CHANGES file

File: cryptofeed/gemini/gemini.py [hunk 2]
```diff
@@ -60,7 +60,7 @@ def __init__(self, pairs=None, channels=None, callbacks=None, **kwargs):
 
     async def _trade(self, msg, timestamp):
         price = Decimal(msg['price'])
-        side = BID if msg['makerSide'] == 'bid' else ASK
+        side = BUY if msg['makerSide'] == 'bid' else SELL
```
Comment (ooinaru) [inline cryptofeed/gemini/gemini.py]: @bmoscon Can you chack this works correctly? I don't have access to Gemini where I am from but it looks to me as if this should be `side = BUY if msg['makerSide'] == 'ask' else SELL`,

Comment (bmoscon) [reply cryptofeed/gemini/gemini.py]: Looks right and matches the docs. I'll give it a test today

**Pull request merged**